modules/ROOT/pages/upgrade-custom-policy-mule3.adoc

```diff
@@ -0,0 +1,35 @@
+= Upgrade Custom Policies for Mule 3
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+In API Manager, you can upgrade the version of your custom policies for your Mule 3 applications.
+
+== Uploading a New Version of Your Custom Policy
+
+. In API Manager, go to *Custom Policies*, and click *Add Custom Policy*.
+. Select the *Policy for Runtimes olden than Mule 4*.
+. In the *Name* field, type the name of your new policy version. For example, `Regex Filter Policy v2`.
+. Upload your YAML and XML files for your custom policy's new version.
++
+image::upgrade-custom-policy-mule3-08395.png[align=center]
+. Click *Add*.
+
+== Upgrading your Custom Policy Version in your API Instance
+
+. In API Manager, go to *API Administration*, and select the version of the API instance you want to upgrade its custom policy.
+. Click *Policies*.
+. Expand the policy name you want to upgrade, click *Actions*, and select *Change Version*.
++
+image::upgrade-custom-policy-mule3-3e0bb.png[align=center]
+. Select the new version of your custom policy, and click *Configure Policy*.
++
+image::upgrade-custom-policy-mule3-0a3c0.png[align=center]
++
+Note that the fields that you configured on your earlier versions are still configured in your upgraded policy. Additionally, the UI points out the new fields to configure.
```

Comment:
Change version of the policy.
UI will show a NEW tag besides new filed.
Current policy Sensitive information will be masked out.